librdmacm/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ publish_headers(rdma
   )
 publish_headers(infiniband
   ib.h
+  endian.h
   )
```

Comment:
I have reworked <librdmacm/endian.h> such that it redefines the glibc macros instead of introducing new conversion functions.